test/pls.jl

```diff
@@ -34,11 +34,17 @@ end
     @test fm1.optsum.initial == ones(1)
     fm1.θ = ones(1)
     @test fm1.θ == ones(1)
+    @test islinear(fm1)
+    @test responsename(fm1) == "yield"
+    @test meanresponse(fm1) ≈ 1527.5
+    @test modelmatrix(fm1) == ones(30, 1)
+    @test weights(fm1) == ones(30)
```

Comment:
can you add a test for a linear model that has non unit weights?

Reply:
Grumble, grumble - yes, okay.